src/datalinker/constants.py

```diff
@@ -9,7 +19,7 @@
 ADQL_IDENTIFIER_REGEX = r"^[a-zA-Z0-9_]+$"
 """ADQL table (without prefix)."""
 
-HIPS_DATASETS = (
+HIPS_DATASETS = [
```

Comment:
I think you can drop this now too in favor of a default in Phalanx and the new configuration.